app/services/reports/docx/repository_helper.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Reports::Docx::RepositoryHelper
+  include InputSanitizeHelper
+
+  def prepare_row_columns(repository_data)
+    result = [repository_data[:headers]]
+    repository_data[:data].repository_rows.each do |record|
+      row = []
+      row.push(record.id)
+      row.push(escape_input(record.name))
+      row.push(I18n.l(record.created_at, format: :full))
+      row.push(escape_input(record.created_by.full_name))
+
+      custom_cells = record.repository_cells
+      repository_data[:data].mappings.each do |column_id, _position|
+        value = custom_cells.find_by(repository_column_id: column_id)&.value&.formatted
```

Comment:
So it loads each cell and value separately?

Reply:
I need check for empty cells and place them in correct order. In `mappings` we have correct column order to match with header.

Reply:
It is not right way to do it as it will generate tons of extra SQL requests

Reply:
Changed